nautobot/extras/migrations/0057_alter_role_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.16 on 2022-12-29 17:18
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0056_alter_configcontext_roles"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="role",
+            options={"ordering": ("weight", "name")},
+        ),
```

Comment:
Does this need to be a separate migration, or can it be merged up into `0054_role_and_alter_status.py`?

Reply:
I tried that but, `makemigration` kept on creating this migration file regardless.